app/templates/base-flat.jade

```diff
@@ -67,6 +67,11 @@ mixin accountLinks
                         a.text-p.text-teal(href="/about", data-i18n="nav.about")
                       else
                         a.text-p(href="/about", data-i18n="nav.about")
+                    li
```

Comment:
Adding this to the nav it's more likely to hit this wrapping issue. Does the nav wrapping issue matter? When the screen gets a little narrower the nav collapses into a hamburger menu so it's only a small range of display.

![Screen Shot 2020-06-15 at 5 47 34 PM](https://user-images.githubusercontent.com/15080861/84719506-666e0a00-af30-11ea-9b0f-bd4f0ea54c54.png)

Reply:
I have a few small changes I need to make this week and I'll include this in those changes.  We're looking at adjusting the font size in response to window size too and I'll take a look at adjusting the navbar size